src/Media_Command.php

```diff
@@ -185,6 +185,9 @@ public function regenerate( $args, $assoc_args = array() ) {
 	 * [--post_name=<post_name>]
 	 * : Name of the post to attach the imported files to.
 	 *
+	 * [--slug=<slug>]
```

Comment:
```suggestion
	 * [--file_name=<name>]
```

Let's use `file_name` as the argument, for greater clarity.

Reply:
DOne